webapp/src/components/workspace.tsx

```diff
@@ -21,16 +24,36 @@ type Props = {
     readonly: boolean
 }
 
-const Workspace = React.memo((props: Props) => {
-    const {boardTree, setSearchText, workspaceTree, showBoard, showView, setLanguage} = props
+function centerContent(props: Props) {
```

Comment:
In fact, this is a functional component, we should make it uppercase and use as a component bellow.